lib/monis_app/finance/transaction.ex

```diff
@@ -0,0 +1,26 @@
+defmodule MonisApp.Finance.Transaction do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+  schema "transactions" do
+    field :comment, :string
+    field :payee, :string
+    field :transaction_date, :date
+    field :value, :integer
```

Comment:
Add `enabled` or `is_active` field to handle soft deletion

Reply:
Does it make sense to soft delete transactions instead of hard deleting? I would expect deleted transactions to be hard deleted as a user